saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -872,10 +925,27 @@ class Meta:
     @classmethod
     @traced_atomic_transaction()
     def perform_mutation(cls, _root, info, **data):
-        manager = load_plugin_manager(info.context)
-        variant = cls.get_node_or_error(
-            info, data["variant_id"], only_type=ProductVariant
+        sku = data.get("sku")
+        variant_id = data.get("variant_id")
+        validate_one_of_args_is_in_mutation(
+            ProductErrorCode, "sku", sku, "variant_id", variant_id
         )
+
+        manager = load_plugin_manager(info.context)
+
+        if variant_id:
+            variant = cls.get_node_or_error(info, variant_id, only_type=ProductVariant)
+        else:
+            variant = models.ProductVariant.objects.filter(sku=sku).first()
```

Comment:
In the database, SKU could be "null". There is a change to perform this line with `sku=None`?

Reply:
`validate_one_of_args_is_in_mutation` will return error in that case